lisa/util/constants.py

```diff
@@ -136,3 +136,36 @@
 DEVICE_TYPE_GPU = "GPU"
 
 DISK_PERFORMANCE_TOOL = "fio"
+
+NTTTCP_TCP_CONNECTIONS = [
```

Comment:
1. Add them into ntttcp. It's ok to import constants from a tool module like "from lisa.tools.ntttcp import NTTTCP_TCP_CONNECTIONS.
2. The name can be more accurate like NTTTCP_TCP_CONCURRENCY.

Reply:
update